xapian-letor/m4-macros/xapianletor.m4

```diff
@@ -0,0 +1,150 @@
+# Get XAPIANLETOR_CXXFLAGS, XAPIANLETOR_LIBS, and XAPIANLETOR_VERSION from xapianletor-config and
+# AC_SUBST() them.
+
+# serial 17
```

Comment:
I'm fairly easy on this, but it seems odd to start a new macro file with a serial of something other than 1.

Reply:
I wondered that, but it's also helpful to minimise pointless differences between this and `xapian.m4`.

Reply:
Can you explain what is this serial number and it's significance?.

Reply:
@VaibhavKansagara check out [the autotools documentation on the subject](https://www.gnu.org/software/automake/manual/html_node/Serials.html).